app/addons/config/components.react.jsx

```diff
@@ -15,8 +15,8 @@ import ReactDOM from "react-dom";
 import Stores from "./stores";
```

Comment:
Was this file supposed to be replaced by `app/addons/config/components.js`?  My guess is yes, since this is still a jsx file.